plugin-graphql importing apollo/client file that no longer exists in recent releases

## Type of Change
Bug

## Summary

When running `greenwood develop` and using `@greenwood/plugin-graphql` in my greenwood.config.js I see:
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/skynet/workspace/greenwood/rubiks/node_modules/@apollo/client/core/core.cjs.js' imported from /home/skynet/workspace/greenwood/rubiks/node_modules/@greenwood/plugin-graphql/src/core/cache.js
```

## Details

@greenwood/plugin-graphql is importing from `apollo/client/core/core.cjs.js` which no longer exists in more recent releases. Instead the file `apollo/client/core/core.cjs` exists without the `.js` extension.

 The yarn.lock and package.json should be updated with the most recent release of @apollo/client as well

--- a/packages/plugin-graphql/package.json
+++ b/packages/plugin-graphql/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@greenwood/plugin-graphql",
-  "version": "0.28.2",
+  "version": "0.28.3",
   "description": "A plugin for using GraphQL for querying your content.",
   "repository": "https://github.com/ProjectEvergreen/greenwood",
   "homepage": "https://github.com/ProjectEvergreen/greenwood/tree/master/packages/plugin-graphql",
@@ -27,7 +27,7 @@
     "@greenwood/cli": "^0.4.0"
   },
   "dependencies": {
-    "@apollo/client": "^3.3.11",
+    "@apollo/client": "^3.7.14",
     "@rollup/plugin-alias": "^3.1.2",
     "apollo-server": "^2.21.0",
     "graphql": "^15.5.0",

--- a/packages/plugin-graphql/package.json
+++ b/packages/plugin-graphql/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@greenwood/plugin-graphql",
-  "version": "0.28.3",
+  "version": "0.28.2",
   "description": "A plugin for using GraphQL for querying your content.",
   "repository": "https://github.com/ProjectEvergreen/greenwood",
   "homepage": "https://github.com/ProjectEvergreen/greenwood/tree/master/packages/plugin-graphql",

--- a/packages/plugin-graphql/src/core/cache.js
+++ b/packages/plugin-graphql/src/core/cache.js
@@ -1,3 +1,5 @@
+// we "deep" link into the Apollo package to avoid pulling in React
+// https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#using-apollo-client-without-react
 import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client/core/index.js';
 import { checkResourceExists } from '@greenwood/cli/src/lib/resource-utils.js';
 import fs from 'fs/promises';

--- a/packages/plugin-graphql/src/core/cache.js
+++ b/packages/plugin-graphql/src/core/cache.js
@@ -1,4 +1,4 @@
-import ApolloCore from '@apollo/client/core/core.cjs.js';
+import ApolloCore from '@apollo/client/core/core.cjs';
 import { checkResourceExists } from '@greenwood/cli/src/lib/resource-utils.js';
 import fs from 'fs/promises';
 import { gql } from 'apollo-server';

--- a/packages/plugin-graphql/src/core/cache.js
+++ b/packages/plugin-graphql/src/core/cache.js
@@ -1,4 +1,4 @@
-import ApolloCore from '@apollo/client/core/core.cjs';
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client/core/index.js';
 import { checkResourceExists } from '@greenwood/cli/src/lib/resource-utils.js';
 import fs from 'fs/promises';
 import { gql } from 'apollo-server';
@@ -9,12 +9,12 @@ const createCache = async (req, context) => {
 
   return new Promise(async(resolve, reject) => {
     try {
-      const client = await new ApolloCore.ApolloClient({
-        link: new ApolloCore.HttpLink({
+      const client = await new ApolloClient({
+        link: new HttpLink({
           uri: 'http://localhost:4000?q=internal', /* internal flag to prevent looping cache on request */
           fetch
         }),
-        cache: new ApolloCore.InMemoryCache()
+        cache: new InMemoryCache()
       });
 
       /* Take the same query from request, and repeat the query for our server side cache */

